style-guide.rst

```diff
@@ -146,6 +146,7 @@ For external links, use one of the following methods.
 
 Link inline:
   Define occasional links directly within the surrounding text.
+  To make the link text show up in code-style (which excludes it from the spelling check), use the custom ``:literalref:`` role (requires the :literalref:`lxd-sphinx-extensions <https://github.com/canonical/lxd-sphinx-extensions>` package).
```

Comment:
How about getting rid of it completely and making it clear that the style guide applies to the starter pack?
There we load all the "custom" extensions, so we don't really need to make a difference.

This might be confusing for people who find this style guide and don't use the starter pack, but I don't think that's our problem. ;-)

Reply:
My take on this is that we recommend using the starter pack and its corresponding style guide (this one here).
If teams want to do something different, they are free to do so, but they need to maintain their own style guide then.